Dockerfile

```diff
@@ -1,13 +1,8 @@
-FROM golang:1.13.10-alpine as builder
-WORKDIR /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure
-ADD . .
 ARG TARGETARCH
 ARG TARGETOS
-ARG LDFLAGS
-RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi-driver-provider-azure ./cmd/
```

Comment:
Moving the binary build out of the container, so we can sign the `.exe` and then load into container